server/cli.js

```diff
@@ -1,5 +1,6 @@
 import {getCommissionReport} from "./reports/commission/commission-endpoint.js";

```

Comment:
BTW, cli.js currently has CRLF line endings instead of LF... at least for me with my git and VS settings. I'll make a note and do this after merging. I like these clean diffs but I also want to deliver consistent file formats.